### PR TITLE
[CU-3383bzz] fix list_assets rpc

### PR DESCRIPTION
**Pull request by andresmechali**
https://app.clickup.com/t/3383bzz

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-3383bzz Statemine USDT - Updating decimals for RPC assets](https://app.clickup.com/t/3383bzz)

**Review by dzmitry-lahoda — COMMENTED** [review]
@benluelo please sync up on this. As you will see in your PR currency.rs was fixed and will have conflict.

File: code/parachain/runtime/primitives/src/currency.rs [hunk 1]
```diff
@@ -89,22 +89,12 @@ macro_rules! list_assets {
...
-			None
-		}
-
 		pub fn list_assets() -> Vec<Asset<XcmAssetLocation>> {
 			[
 				$(Asset {
 					id: CurrencyId::$NAME.0 as u128,
 					name: Some(stringify!($NAME).as_bytes().to_vec()),
-					decimals: 12_u8.into(),
+					decimals: Self::decimals(),
```
Comment (dzmitry-lahoda) [inline code/parachain/runtime/primitives/src/currency.rs:97]: not sure if we can break UI now, but better assumption could be if decimals are made Option

Comment (dzmitry-lahoda) [reply code/parachain/runtime/primitives/src/currency.rs:97]: but that just wish, very low prio and not blocker.

Comment (andresmechali) [reply code/parachain/runtime/primitives/src/currency.rs:97]: As far as I understand, we are not planning to include assets that don't have decimals. At least not for now.

**Review by PoisonPhang — APPROVED** [review]

**Review by Dom-Roth — APPROVED** [review]

**Pull request closed without merging**